scripts/upgrade

```diff
@@ -1,14 +1,232 @@
-#!/bin/bash                                                                                                                                                                                      
+#!/bin/bash
```

Comment:
No metronome update in this file?

Reply:
As the actual package is 5 years old, I considered that there is no jitsi_ynh installed, so no need to upgrade metronome configuration from actual in production jitsi_ynh

In addition current package don't have a `version` in `manifest.json`, so if someone has a current jitsi_ynh installed, would be better to remove it and reinstall from scratch.

Reply:
If the config file changes in the future, we have to be careful with that :-)